lib/Dancer2/Core/App.pm

```diff
@@ -833,6 +833,41 @@ sub log {
     $logger->$level(@_);
 }
 
+sub send_as {
+    my $self = shift;
+    my ( $type, $data, $options ) = @_;
+    $options ||= {};
+
+    # allow lower cased serializer names
+    my $prefix = 'Dancer2::Serializer::';
+    my $serializer_class;
+    for ( uc $type, $type ) {
```

Comment:
That wouldn't necessarily work either because our capitalization rules allow you to have "SerealButNotThePodcast" using underscores. I would suggest using the capitalization code we have[1](https://github.com/PerlDancer/Dancer2/blob/master/lib/Dancer2/Core.pm#L7), but that won't work for "JSON".

So now I have two problems:
* I don't want to make a special case for "JSON" and "YAML" which will block the other options. (Does it block other options? Where do you specify the correct casing?)
* I don't want to allow "json" because it's not the right one. "JSON" is the right one.

(Regarding the latter option, one can suggest we support everything in lowercase, but since we need to match it to package names, it becomes more of a mess because you *do* need to know the correct casing for the package to use it, so it will be a mess to handle.)